README.md

```diff
@@ -16,23 +16,21 @@ Currently unsorted, the following is intended to function as a discussion guide
 to developers/contributors.
 
 ### In Progress
```

Comment:
`tentative roadmap` maybe? Or something similar to that. It's not really a task, but a group of tasks